src/TLuaInterpreter.cpp

```diff
@@ -750,21 +757,220 @@ int TLuaInterpreter::getBufferTable( lua_State * L )
 
 int TLuaInterpreter::loadRawFile( lua_State * L )
 {
-    string luaSendText="";
-    if( ! lua_isstring( L, 1 ) )
-    {
-        lua_pushstring( L, "loadRawFile: wrong argument type" );
+    QString fileName;
+    if( ! lua_isstring( L, 1 ) ) {
+        lua_pushstring( L, tr( "loadRawFile: bad argument #1 (replay file, optionally with a path, as string expected, got %1)" )
+                           .arg( luaL_typename(L, 1) ).toUtf8().constData() );
         lua_error( L );
         return 1;
     }
     else
-    {
-        luaSendText = lua_tostring( L, 1 );
+        fileName = QDir::fromNativeSeparators( QString::fromUtf8( lua_tostring( L, 1 ) ) );
+
+    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
+#if defined( TILDE_EXPANSION )
```

Comment:
Yes, but we might as well not make it harder for scripts and force them to use ~ or something else - having ~ work everywhere will make scripts not require if/else conditions.